src/main/java/org/jenkinsci/plugins/ParameterizedRemoteTrigger/RemoteJenkinsServer.java

```diff
@@ -78,6 +80,10 @@ public boolean getHasBuildTokenRootSupport() {
         return this.hasBuildTokenRootSupport;
     }
 
+    public boolean hasCrumbSupport() {
```

Comment:
This should be renamed to 'getHasCrumbSupport', otherwise the config page will not pick up the current value.